### DISCUSSION TITLE
Error: Process completed with exit code 1.

### DISCUSSION THREAD

**Discussion by kristenlui** [alshedivat/al-folio#2150] [Q&A] [answered]
After deleting a file in the "post" folder (a distill-style blog post) yesterday, I found myself stuck in a continuous error loop and unable to modify the webpage content. I am completely puzzled by my own mistake, as there were no issues with modifications before. Please help me identify the problem.

Error: Process completed with exit code 1.

Liquid Exception: Zero vectors can not be normalized in /home/runner/work/kristenlui.github.io/kristenlui.github.io/_layouts/post.html 
463 ------------------------------------------------ 
464 Jekyll 4.3.3 Please append `--trace` to the `build` command 
465 for any additional information or backtrace. 
466 ------------------------------------------------

**Comment by george-gca** [comment, marked as answer]
Check our [FAQ about this information](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#my-site-doesnt-work-when-i-enable-related_blog_posts-why).
